src/ImageSharp/Formats/Qoi/QoiConstants.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Six Labors.
+// Licensed under the Six Labors Split License.
+
+using System.Text;
+
+namespace SixLabors.ImageSharp.Formats.Qoi;
+
+internal static class QoiConstants
+{
+    /// <summary>
+    /// Gets the bytes that indicates the image is QOI
+    /// </summary>
+    public static ReadOnlySpan<byte> Magic => Encoding.UTF8.GetBytes("qoif");
```

Comment:
This property calls `GetBytes` (and allocates) on every invocation.
For .NET 7 this could be written as
```suggestion
    public static ReadOnlySpan<byte> Magic => "qoif"u8;
```
and before as
```suggestion
    private static readonly byte[] s_magic = Encoding.UTF8.GetBytes("qoif");
    public static ReadOnlySpan<byte> Magic => s_magic;
```

Reply:
We're using .NET 6 as far as I know so I'll go with the second option